roles/openshift_openstack/tasks/provision.yml

```diff
@@ -21,5 +21,11 @@
   when:
   - openshift_openstack_stack_state == 'present'
 
-# TODO(shadower): create the registry and PV Cinder volumes if specified
-# and include the `prepare-and-format-cinder-volume` tasks to set it up
+- name: Create the Cinder volume for OpenShift Registry
+  include_tasks: create-registry-volume.yml
```

Comment:
Is the prepare-and-format-cinder-volume not a thing any more?

Reply:
Nope, turns out it's no longer necessary. OpenShift will format the volumes as needed.